content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -438,15 +438,15 @@ Figure 12
 
 6. Click Finish
 
-## **Execute the Application**
+## Execute the Application
 
 Lastly, you can use Postman to send a Http Get request to the endpoint to retrieve the stored secret as shown below:
 
 ![](media/383d2af1fcf9e2a16f2960de147ca549.png)
 
 Figure 13
 
-## **Summary**
+## Summary
 
 Azure key vault helps you to keep your application's secrets out of the application. You can use it to isolate secrets from your code files. These secrets include connection strings, API keys, environment variables, etc. You can take advantage of Azure Key Vault to keep secrets out of source control or out of your application in a centralized storage place. In this article we've taken advantage of a managed identity to connect an Azure web app in .NET to an Azure Key Vault and retrieve secret value from there.
```

Comment:
Changes: In this article`,` we've `used` managed identity to connect an Azure web app in .NET to Azure Key Vault and retrieve secret value from there.

Reply:
Fixed, thanks!